packages/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -72,19 +72,14 @@ const insertBandwidthLimit = (sdpLines, index) => {
  */
 const setRemoteVideoConstraints = (sdp, level = QUALITY_LEVELS.HIGH) => {
   const maxFs = REMOTE_VIDEO_CONSTRAINTS.MAX_FS[level];
-  const maxMbps = REMOTE_VIDEO_CONSTRAINTS.MAX_MBPS[level];
 
-  if (!maxFs || !maxMbps) {
+  if (!maxFs) {
     throw new ParameterError(`setRemoteVideoConstraints: unable to set max ${!maxFs ? 'framesize' : 'mbps'}, value for level "${level}" is not defined`);
```

Comment:
Error message should be changed

Reply:
@rajeshtezu90, We can remove the mbps from error description. Thanks.

Reply:
Done. Thanks @rajeshtezu90.